docs/src/modules/components/ad.styles.js

```diff
@@ -5,7 +5,8 @@ const adBodyImageStyles = (theme) => ({
   root: {
     display: 'block',
     overflow: 'hidden',
-    border: `1px solid ${alpha(theme.palette.action.active, 0.12)}`,
+    border: '1px dashed',
```

Comment:
I think it could be softer too, but I'm not so certain about the dashed line. Couldn't it just be a softer gray instead?

Reply:
For sure it could ⎯ thought that the dashed border gave more of a feeling that the ads are "attached" to the UI rather than part of it. Sort of became less distracting, to me, I think... 🤔

Reply:
It makes sense. It's more an opinion rather than anything else—just not a big fan of dashed lines 😅 
What if it was dotted instead? I think it ends up being more regular than dashed lines, and even more subtle.

Reply:
Dotted ends up being too small 😢